slices/git.yaml

```diff
@@ -0,0 +1,41 @@
+package: git
+
+essential:
+  - git_copyright
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - libpcre2-8-0_libs
+      - zlib1g_libs
+    contents:
+      /usr/bin/git:
+      /usr/bin/git-shell:
+      /usr/bin/scalar:
+
+  core:
+    essential:
+      - libc6_libs
+      - libcurl3t64-gnutls_libs
+      - libexpat1_libs
+      - libpcre2-8-0_libs
+      - zlib1g_libs
+      - git_bins
+    contents:
+      /usr/lib/git-core/**:
+
+  contrib:
```

Comment:
What's the usage of the `contrib` slice?

Reply:
That is a good point. I thought somehow it was relevant because of the scripts in that directory, but it looks like the source code here needs to be built first. I will exclude it, which also fixes some broken symlinks you pointed out.